packages/contentstack-migration/package.json

```diff
@@ -1,14 +1,15 @@
 {
   "name": "@contentstack/cli-migration",
-  "version": "1.1.1",
+  "version": "1.2.0",
   "author": "@contentstack",
   "bugs": "https://github.com/contentstack/cli/issues",
   "dependencies": {
-    "@contentstack/cli-command": "^1.1.0",
-    "@contentstack/cli-utilities": "^1.1.0",
+    "@contentstack/cli-command": "^1.2.0",
+    "@contentstack/cli-utilities": "^1.2.0",
     "@contentstack/management": "^1.6.1",
     "@oclif/command": "^1.8.16",
     "@oclif/config": "^1.18.3",
+    "@oclif/core": "^2.3.0",
```

Comment:
@aman19K we can remove this package if not used directly

Reply:
@antonyagustine removed @oclif/core dependency from package.json